nifi-assembly/LICENSE

```diff
@@ -2073,3 +2073,93 @@ style license.
    WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
    OTHER DEALINGS IN THE SOFTWARE.
+
+  The binary distribution of this product bundles 'ANTLR 4' which is available
+    under a "3-clause BSD" license.  For details see http://www.antlr.org/license.html
+
+      Copyright (c) 2012 Terence Parr and Sam Harwell
+      All rights reserved.
+      Redistribution and use in source and binary forms, with or without modification, are permitted
+      provided that the following conditions are met:
+
+      Redistributions of source code must retain the above copyright notice, this list of
+      conditions and the following disclaimer.
+      Redistributions in binary form must reproduce the above copyright notice, this list of
+      conditions and the following disclaimer in the documentation and/or other materials
+      provided with the distribution.
+
+      Neither the name of the author nor the names of its contributors may be used to endorse
+      or promote products derived from this software without specific prior written permission.
+
+      THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+      EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+      MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+      THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+      SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+      PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+      INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+      STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
+      THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+  The binary distribution of this product bundles 'icu4j'
+    which is available under a X-style license.
```

Comment:
Cannot have 'X-Style'.  Is this MIT?  Can you share the link to the LICENSE this is coming from?

Reply:
The reference to "X-Style" comes from [this page](http://source.icu-project.org/repos/icu/icu4j/tags/release-4-8-1/readme.html#license). However the [license](http://source.icu-project.org/repos/icu/icu4j/tags/release-4-8-1/main/shared/licenses/license.html) does look MIT-style.

Reply:
Please Refer to it as the ICU License as noted here http://source.icu-project.org/repos/icu/icu4j/tags/release-4-8-1/main/shared/licenses/license.html.

Fortunately that is in the Apache Category A so we're good here.  Just call it out as the ICU License as in the LIcense text instead of 'X-Style'

thanks

Reply:
Done, and squashed to 2 commits (original from Vadim and one of mine)